src/topology/locally_constant/basic.lean

```diff
@@ -426,4 +426,41 @@ lemma coe_desc {X α β : Type*} [topological_space X] (f : X → α) (g : α 
 
 end desc
 
+section indicator
+variables {R : Type*} [has_zero R] {U : set X} (f : locally_constant X R)
+
+/-- Given a clopen set `U` and a locally constant function `f`, `locally_constant.indicator`
+  returns the locally constant function that is `f` on `U` and `0` otherwise. -/
+@[simps?]
```

Comment:
`simps?` prints some diagnostics, it shouldn't remain in the final version. Please use something like

* `simps`, or
* `simps {fully_applied := ff}` if you want to get rid of the `(ᾰ : X)`